src/java.desktop/share/classes/sun/java2d/pipe/SpanShapeRenderer.java

```diff
@@ -42,7 +42,6 @@
  * perform the actual rendering.
  */
 public abstract class SpanShapeRenderer implements ShapeDrawPipe {
-    static final RenderingEngine RenderEngine = RenderingEngine.getInstance();
 
```

Comment:
I suppose this is just clean up when you happened to notice it wasn't used/

Reply:
Exactly.